KnockKnock-iOS/Scenes/Login/LoginInteractor.swift

```diff
@@ -23,54 +27,74 @@ final class LoginInteractor: LoginInteractorProtocol {
   var worker: LoginWorkerProtocol?
   var router: LoginRouterProtocol?
 
-  func fetchLoginResult(
-    source: LoginViewProtocol,
-    socialType: SocialType
-  ) {
+  func fetchLoginResult(socialType: SocialType) {
+
     self.worker?.fetchLoginResult(
+      appleLoginResultDelegate: self,
       socialType: socialType,
-      completionHandler: { loginResponse, loginInfo in
-
-        // 회원 판별
-        // 회원 o -> 토큰 저장 후 홈 화면 진입 / 회원 x -> 프로필 설정화면(회원가입)
-        if loginResponse.isExistUser {
-          self.saveTokens(loginResponse: loginResponse)
-          self.popLoginView(source: source)
-          NotificationCenter.default.post(name: .loginCompleted, object: nil)
-        } else {
-          self.navigateToProfileSettingView(
-            source: source,
-            loginInfo: loginInfo
-          )
-        }
+      completionHandler: { response, loginInfo in
+
+        self.isExistUser(
+          response: response,
+          loginInfo: loginInfo
+        )
       }
     )
   }
+  
+  /// 회원 판별
+  /// isExisted: 기존 회원 여부
+  /// 회원 o -> 토큰 저장 후 홈 화면 진입 / 회원 x -> 프로필 설정화면(회원가입)
```

Comment:
```suggestion
  /// 회원 판별
  ///
  /// - Parameters:
  ///   - isExisted: 기존회원 여부
  ///   - loginInfo: 로그인 요청 정보
```

이렇게 작성하면 method summary에 나오게 됩니다 😁  xcode 에서 확인해보셔요